hybrid/SampleApps/NoteSync/NoteSync/SFContentSoqlSyncDownTarget.m

```diff
@@ -225,40 +225,16 @@ @interface SFContentSoqlSyncDownTarget ()
 
 @implementation SFContentSoqlSyncDownTarget
 
-- (instancetype)initWithDict:(NSDictionary *)dict {
```

Comment:
Setting the type is not needed since the super class sets the implementation class and newFromDict looks for the implementation class first.